packages/loader/container-loader/closeAndGetPendingLocalState.md

```diff
@@ -0,0 +1,31 @@
+# Stashed Changes
+
+When a user goes offline, they are still able to make changes to the document. When the user goes back online, the DDSs will be asked to resubmit the ops, which will use the merging strategy of the DDS to modify the ops on top of new ops so they can be resubmitted and understood by other clients. If `Container.close()` is called while still offline, the changes will be lost. If closed while online, but the changes have not been sequenced yet (i.e. the container is "dirty"), the changes may or may not be lost.
```

Comment:
We're not super consistent about this, but I'm a fan of inserting single-line breaks on natural sentence boundaries (periods usually, commas sometimes) in markdown files. This gives easier to review `git diffs` for future changes to the document without hindering readability in other contexts.